### PR TITLE
publishing workflow name along with task name.

### PR DESCRIPTION
**Pull request by manan164**

### CODE REVIEW

File: core/src/main/java/com/netflix/conductor/core/execution/ArchiveWorkflowViaReminderStatusListener.java [hunk 1]
```diff
@@ -41,34 +41,27 @@ private void initialize() {
...
 
     @Override
     public void onWorkflowTerminated(Workflow workflow) {
         LOG.debug("Workflow {} is terminated", workflow.getWorkflowId());
-        archiveWorkflow(workflow);
+        archiveWorkflow(workflow.getWorkflowId());
     }
 
-    public void archiveWorkflow(Workflow workflow) {
+    public void archiveWorkflow(String workflowId) {
```
Comment (rksinha) [inline core/src/main/java/com/netflix/conductor/core/execution/ArchiveWorkflowViaReminderStatusListener.java:53]: Any benefit of doing this? Test cases will also have to change.

File: core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java [hunk 2]
```diff
@@ -903,8 +901,8 @@ public void updateTask(TaskResult taskResult) {
         if (task.getStatus().isTerminal()) {
             long duration = getTaskDuration(0, task);
             long lastDuration = task.getEndTime() - task.getStartTime();
-            Monitors.recordTaskExecutionTime(task.getTaskDefName(), duration, true, task.getStatus());
-            Monitors.recordTaskExecutionTime(task.getTaskDefName(), lastDuration, false, task.getStatus());
+            Monitors.recordTaskExecutionTime(task.getTaskDefName(), duration, true, task.getStatus(), task.getWorkflowType());
```
Comment (rksinha) [inline core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java:904]: getWorkflowType is the workflow name?

File: core/src/main/java/com/netflix/conductor/core/execution/ArchiveWorkflowViaReminderStatusListener.java (hunk 1 above)
Comment (manan164) [reply core/src/main/java/com/netflix/conductor/core/execution/ArchiveWorkflowViaReminderStatusListener.java:53]: We were updating the workflow output to debug one of the issues. Now, this is not needed. Test cases. ar already. updated.

File: core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java (hunk 2 above)
Comment (manan164) [reply core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java:904]: Yeah

**Pull request merged**